Design hot fixes

DevStarlight opened the pull request:
Full redesign of [#156613344](https://www.pivotaltracker.com/story/show/156613344)

DevStarlight committed:
fixed upload button in topbar, not having the right color.
svg refactor of the button

alvaromart committed:
video-detail-view.html Added class card-video-detail to section, added padding in css to show more space. Video author, remove title and make value strong. video-detail-show-info, added ng-init showDetails = true.
video-detail-view.css. Added cursor pointer over card-user-info-desktop, navigate to user videos. Added card-video-detail and padding to show more space in card.

DevStarlight committed:
- removed unnecesary directive params (width and height)
- added a container to the image so it keeps heights uniformed.
- added moment library to manage dates
- fixed grouping in checked svg
- if no product type, don't show anything
- Fixed videoList sizes

DevStarlight committed:
Allow to download the video in case you are the owner

DevStarlight committed:
Allow register, even if you are logged in already

DevStarlight committed:
Home screen:
  -  Include app store & google play icons in desktop
  -  Reduce logo size

DevStarlight committed:
Fix login design issues
  -  Quitar olvidar contraseña hasta que esté la historia.
  -  Mensajes de error, en caja debajo.

DevStarlight committed:
Changes in upload video screen:
  -  Increase the number of rows in the description field
  -  Added border-radius to upload button
  -  Adjust left-preview height size to 16:9 format

CodingCarlos commented on pages/video-download/video-download.view.html:
```diff
@@ -8,8 +8,10 @@
 
 	<div ng-if="VideoDownload.video.data.error && !VideoDownload.loading" class="headers-info-videoDownload">
+			<!-- TO-DO: Set translation -->
 		<h1 class="header">Video erróneo</h1>
```
Don't forget translating this ;)

DevStarlight replied:
holy sh**, thanks!

CodingCarlos commented on pages/video-download/video-download.view.html:
```diff
@@ -8,8 +8,10 @@
 		<h1 class="header">Video erróneo</h1>
 		<div>
+			<!-- TO-DO: Set translation -->
 			No hemos podido encontrar ese video. Por favor, comprueba que el enlace es correcto, y prueba de nuevo.
```
Don't forget translating this ;)

CodingCarlos reviewed (approved):
Remember translating that two things, and my corde review is done ;)

alvaromart committed:
video-card.css added line-heigh: normal, fix firefox text razored.
video-card.view.html translate key product type to correct language
video-download.view, removed TODO translate.
languages/en_us, es_es added ERROR_DOWNLOAD_VIDEO, ERROR_DOWNLOAD_VIDEO_NOT_EXISTS

DevStarlight committed:
Fix an issue at moment library:
  -  Added languaje support for moment
  -  Fix an issue at moment, capitalize the first letter

DevStarlight committed:
Merge conflicts in video-download.view.html

DevStarlight committed:
fixed an issue with a wrong height size
removed unnecesary max-height since we can fit the same size with % and not with vw

DevStarlight committed:
fixed no user image
fixed no user position

DevStarlight committed:
fix upload div size

DevStarlight committed:
Merge pull request #48 from Videona/156613344_fix_video_details

[156613344] Hot fixed video details.

DevStarlight committed:
setup 0.5.0:
Improve images, colors, sizes & other stuff.
Ability to download a video if you are the owner.
Ability to register being logged in.
